syntax = "proto3";

package tateyama.proto.framework.request;

option java_package = "com.nautilus_technologies.tateyama.proto";
option java_outer_classname = "FrameworkResponseProtos";

// common response header for client to receive
message Header {
  // the request message version.
  uint32 message_version = 1;

  // reserved for system use
  reserved 2 to 10;

  // session ID.
  uint32 session_id = 11;
}